package.json

```diff
@@ -26,15 +26,15 @@
         "localized-countries": "^2.0.0",
         "lucene-escape-query": "^1.0.1",
         "mjolnir.js": "^2.4.0",
-        "mui-nested-menu": "^3.0.0",
+        "mui-nested-menu": "^3.2.1",
```

Comment:
please keep the form ^3.0.0 in here, even if you should update the dep in package-lock